pkg/manifests/types.go

```diff
@@ -21,6 +21,7 @@ import (
 
 type CollectionProfile string
 type CollectionProfiles []CollectionProfile
+type ExternalLabels map[string]string
```

Comment:
capitalized mainly to distinguish it from the filed name in the docs.

Reply:
needs a doc entry like https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/monitoring/config-map-reference-for-the-cluster-monitoring-operator#tlsconfig